docs/docs/components/pressable.mdx

```diff
@@ -0,0 +1,154 @@
+---
+id: pressable
+title: Pressable
+sidebar_label: Pressable
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import GifGallery from '@site/components/GifGallery';
+
+<GifGallery>
+  <img src={useBaseUrl('gifs/pressable.gif')} width="70%" />
+</GifGallery>
+
+:::info
+This component is a drop-in replacement for the `Pressable` component.
+:::
+
+`Pressable` is a component that can detect various stages of tap, press and hover interactions on any of its defined children.
+
+### Usage:
+
+To use `Pressable`, import it in the following way:
+
+```js
+import { Pressable } from 'react-native-gesture-handler';
+```
+
+## Properties
+
+### `children`
+
+either children or a render prop that receives a boolean reflecting whether
+the component is currently pressed.
+
+### `style`
+
+either view styles or a function that receives a boolean reflecting whether
+the component is currently pressed and returns view styles.
+
+### `onPress`
+
+called after `onPressOut` when a single tap gesture is detected.
+
+### `onPressIn`
+
+called before `onPress` when a touch is engaged.
+
+### `onPressOut`
+
+called before `onPress` when a touch is released.
+
+### `onLongPress`
+
+called immidiately after pointer has been down for at least `delayLongPress` milliseconds (`500` ms by default).
+
+after `onLongPress` has been called, `onPressOut` will be called as soon as the pointer is lifted and `onPress` will no longer be called at all.
```

Comment:
```suggestion
after `onLongPress` has been called, `onPressOut` will be called as soon as the pointer is lifted, and `onPress` will not be called.
```

Reply:
I'd prefer something more like:
```suggestion
after `onLongPress` has been called, `onPressOut` will be called as soon as the pointer is lifted and `onPress` will no longer be called until the end of the gesture.
```

I think a simple `will not be called.` may be confusing - can be interpreted as `will not be called at this moment`, but what should be implied here is that as soon as `onLongPress` gets called, `onPress` will no longer have a chance to be called until the person lifts all their fingers and placed them down again.

Reply:
I see, `will not be called at all` then?

Reply:
Sure, that sounds the best out of all of them